homeassistant/components/luxtronik/__init__.py

```diff
@@ -0,0 +1,117 @@
+"""Support for Luxtronik heatpump controllers."""
+from datetime import timedelta
+import logging
+
+from luxtronik import LOGGER as LuxLogger, Luxtronik as Lux
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST, CONF_PORT
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+
+LuxLogger.setLevel(level="WARNING")
+
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_PARAMETER = "parameter"
+ATTR_VALUE = "value"
+
+CONF_SENSORS = "sensors"
+CONF_ID = "id"
```

Comment:
Import these constants from const.py

Reply:
@Bouni could you also address this comment please? Thanks :)

Reply:
I've just removed the unsed `CONF_ID ` and `CONF_SENSORS` constants. That was what you meant, right?

Reply:
Yes, thank you.